.github/workflows/stale-issue-add-needs-testing.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     - uses: actions/stale@v3
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-issue-message: "Hi,\nThis issue has gone 180 days without any activity. This means it's time for a check-in to make sure it is relevant today. If you are still experiencing this issue, you can help the project by responding to confirm the problem and by providing any updated reproduction steps.\nThanks for helping out."
+        stale-issue-message: "Hi,\nThis issue has gone 180 days without any activity. This means it is time for a check-in to make sure it is still relevant. If you are still experiencing this issue, you can help the project by responding to confirm the problem and by providing any updated reproduction steps.\nThanks for helping out."
```

Comment:
I might add in: "If you are still experiencing the issue with the latest versions" (to keep it short leaving off "versions of Gutenberg and WordPress").

Reply:
:+1: Added